run-integration-tests.py

```diff
@@ -187,6 +187,46 @@ def run_dynamodb_logstore_integration_tests(root_dir, version, test_name, extra_
             print("Failed DynamoDB logstore integration tests tests in %s" % (test_file))
             raise
 
+def run_dynamodb_commit_owner_integration_tests(root_dir, version, test_name, extra_maven_repo, conf, use_local):
+    print(
+        "\n\n##### Running DynamoDB Commit Owner integration tests on version %s #####" % str(version)
+    )
+    clear_artifact_cache()
+    if use_local:
+        run_cmd(["build/sbt", "publishM2"])
+
+    test_dir = path.join(root_dir, \
+        path.join("spark", "src", "main", "java", "io", "delta", "dynamodbcommitstore", "integration_tests"))
+    test_files = [path.join(test_dir, f) for f in os.listdir(test_dir)
+                  if path.isfile(path.join(test_dir, f)) and
+                  f.endswith(".py") and not f.startswith("_")]
+
+    python_root_dir = path.join(root_dir, "python")
+    extra_class_path = path.join(python_root_dir, path.join("delta", "testing"))
+    packages = "io.delta:delta-%s_2.12:%s" % (get_artifact_name(version), version)
+    packages += "," + "org.apache.hadoop:hadoop-aws:3.3.4"
```

Comment:
please help me understand, why do we need this while the test above doesn't?

in logstore integration test we use 3.3.1, how is the version decided? https://github.com/delta-io/delta/blob/master/storage-s3-dynamodb/integration_tests/dynamodb_logstore.py#L64

Reply:
That is a good question. I think just like `dynamodb_logstore.py`, we should let the user specify the version for this library.  

Hi @scottsand-db, this integration test is very similar to https://github.com/delta-io/delta/blob/master/storage-s3-dynamodb/integration_tests/dynamodb_logstore.py that you worked on. One difference is that dynamodb-commitowner is built as part of delta-spark. However, I couldn't get the package to work correctly without adding `org.apache.hadoop:hadoop-aws:3.3.4` as an extra dependency. Should we add `org.apache.hadoop:hadoop-aws:3.3.4` as a direct dependency of `io.delta:delta-*` in build.sbt?